block-storage-attacher/tests/e2e/e2e.sh

```diff
@@ -30,6 +30,10 @@ SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"/scripts
 if [[ $TEST_BLUEMIX_LOGIN == "true" ]]; then
         echo "Bluemix Login DOne"
 	bx_login
+        if [ $ARMADA_REGION == "us-south" ]; then
```

Comment:
can you explain what this check does specific to us-south? how its handled for other regions.

Reply:
This is required for staging only